foxy/distribution.yaml

```
@@ -6974,6 +6974,21 @@ repositories:
       url: https://github.com/lagadic/visp.git
       version: master
     status: maintained
+  vrpn:
+    doc:
+      type: git
+      url: https://github.com/vrpn/vrpn.git
+      version: master
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/alvinsunyixiao/vrpn-release.git
```

@alvinsunyixiao with https://github.com/ros2-gbp/ros2-gbp-github-org/pull/80 merged and deployed, I've mirrored the release repository to https://github.com/ros2-gbp/vrpn-release

```suggestion
      url: https://github.com/ros2-gbp/vrpn-release.git
```

I'd suggest using the ros2-gbp repository for all future releases of all ROS 2 distributions. What do you think?